scripts/intfstat

```diff
@@ -305,6 +305,19 @@ def main():
 
     if save_fresh_stats:
         try:
+            # Add the information also to the general file - i.e. without the tag name
+            if tag_name != '' and tag_name in cnstat_fqn_file.split('/')[-1]:
+                gen_index = cnstat_fqn_file.rfind('/')
+                cnstat_fqn_general_file = cnstat_fqn_file[:gen_index] + cnstat_fqn_file[gen_index:].split('-')[0]
+                if os.path.isfile(cnstat_fqn_general_file):
+                    try:
+                        data = pickle.load(open(cnstat_fqn_general_file, 'rb'))
+                        for key, val in cnstat_dict.items():
+                            data[key] = val
+                        pickle.dump(data, open(cnstat_fqn_general_file, 'wb'))
+                    except IOError as e:
+                        sys.exit(e.errno)
+            # Add the information also to tag specific file
```

Comment:
I think this logic will be an issue. Let's assume the user gives clear stats for globally first and then an individual interface (Ethernet0). First logic it will save the stats of all the interfaces in the file since cnstat_dict will contain information on all interfaces. However, second time when interface name is specified, it cnstat_dict will contain the interface stats alone and writing it directly to the file will overwrite previous written information. As done for 'tag' the logic here when clearing per interface should read the contents of the pickle file and then override only the specific interface.

Reply:
Done